src/index.html

```diff
@@ -1,14 +1,35 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.google.com/specimen/Roboto" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="nav">
+      <div class="nav-wraper">
+        <div class="nav-logo">
+          <a href="#"><img src="images/logo.png" alt="logo" ></a>
+        </div>
+        <nav class="nav-nav">
+          <ul class="nav-list">
+            <li class="nav-item"><a href="#" class="nav-link is-active">Apple</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">Samsung</a></li>
+            <li class="nav-item"><a href="#" class="nav-link">Smartphones</a></li>
+            <li data-qa="hover" class="nav-item"><a href="#" class="nav-link">Laptops & Computers</a></li>
```

Comment:
1. Add empty lines between multiline sibling blocks of HTML
2. Keep your attributes correctly formatted

check examples [here](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting)